test/steps/configuration_steps.go

```diff
@@ -54,12 +54,12 @@ func ConfigurationSteps(suite *godog.Suite, fs *FeatureState) {
 		return nil
 	})
 
-	suite.Step(`^the global new-branch-push-flag configuration is set to true$`, func() error {
+	suite.Step(`^the global new-branch-push-flag configuration is true$`, func() error {
```

Comment:
The global config steps should probably be joined similar to the local config

Reply:
Good catch. I am implementing your suggestion in #1370. In general, I want to keep step implementations much simpler in the new Go version than what we have in Ruby. In particular, I do a lot of full-text searches for step implementations and would like better support for this. That's why I keep the Gherkin simpler and keep step implementations more separate than in the Ruby version.